README.md

```diff
@@ -38,7 +43,14 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```Scope is where a variable can be refrenced. ex.
+var color = blue
+
+function {
+  return color;
+};
+
+the variable [color] is in global scope. and within the function, it's scope is local to the function.
 ```
 
```

Comment:
Correct, but would like to see you comment on where a local variable can be accessed from. What if `color` was defined inside of the function - where can you reference it from?